.github/workflows/release_sdk.yml

```diff
@@ -30,35 +31,42 @@ jobs:
                 version-file: ./sdk/version.go
                 schema: major.minor.build
 
+            - name: Set tag candidate
+              id: set_tag
+              run: |
+                new_tag=sdk/v${{ steps.current_version.outputs.version }}
+                echo $new_tag
+                echo "new_tag=$new_tag" >> "$GITHUB_OUTPUT"
+
             - name: Version check result
-              if: steps.latest_tag.outputs.tag == steps.current_version.outputs.version
+              if: ${{ steps.latest_tag.outputs.tag }} == ${{ steps.set_tag.outputs.new_tag }}
               uses: actions/github-script@v3
               with:
                   script: |
-                      core.setFailed('Version is not updated')
+                      core.setFailed('Version is not updated - please bump on `version.go`')
 
-            # - name: Notify Slack
-            #   if: ${{ success() }}
-            #   uses: rtCamp/action-slack-notify@v2
-            #   env:
-            #       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-            #       SLACK_USERNAME: heliosphere-ci-bot
-            #       SLACK_TITLE: A new Go SDK version was successfully released!
-            #       SLACK_MESSAGE: Version ${{ steps.package-version.outputs.current-version }}
-            #       SLACK_COLOR: ${{ job.status }}
+            - name: Notify Slack
```

Comment:
Shouldn't this be the last step?

Reply:
this is the same order we're doing it everywhere ;) 
wrong of course
fixed it ✌️